src/NSwag.Sample.NET70Minimal/GeneratedClientsCs.gen

```diff
@@ -258,7 +282,9 @@ namespace MyNamespace
                     var disposeResponse_ = true;
                     try
                     {
-                        var headers_ = System.Linq.Enumerable.ToDictionary(response_.Headers, h_ => h_.Key, h_ => h_.Value);
+                        var headers_ = new System.Collections.Generic.Dictionary<string, System.Collections.Generic.IEnumerable<string>>();
```

Comment:
I think this part could just be written as `new System.Collections.Generic.Dictionary<string, System.Collections.Generic.IEnumerable<string>>(response_.Headers);`